### PR TITLE
revise default config

### PR DESCRIPTION
**Pull request by mrogers950**
This trims the config down to specific excludes (discussed in https://github.com/mrogers950/file-integrity-operator/issues/15 ) and include everything in /etc.
@JAORMX @jhrozek

### CODE REVIEW

**Comment by mrogers950** [comment]
I haven't tested this yet (installing new cluster)

File: pkg/controller/fileintegrity/config_defaults.go
Comment (JAORMX) [inline pkg/controller/fileintegrity/config_defaults.go]: Do we want this? probably RHCOS won't have X installed

File: pkg/controller/fileintegrity/config_defaults.go
Comment (JAORMX) [inline pkg/controller/fileintegrity/config_defaults.go:57]: What else is in /opt?

Comment (mrogers950) [reply pkg/controller/fileintegrity/config_defaults.go:57]: Only the CNI plugins, but I suspect UPI systems could have stuff we don't control in there as well.

File: pkg/controller/fileintegrity/config_defaults.go
Comment (mrogers950) [reply pkg/controller/fileintegrity/config_defaults.go]: I'll check, but I left that in from the default. Doesn't hurt if its there though.

File: pkg/controller/fileintegrity/config_defaults.go
Comment (JAORMX) [reply pkg/controller/fileintegrity/config_defaults.go:57]: OK, it makes sense to exclude this then.

**Review by JAORMX — APPROVED** [review]
I'm cool with these changes. Waiting for @jhrozek's feedback before merging.

**Comment by jhrozek** [comment]
Sorry for the delay, for some reason I thought you already merged the changes.
ACK :) I think it is better to catch all and then exclude if there's anything noisy rather than be surprised that something important is not tracked.

**Review by jhrozek — APPROVED** [review]

**Pull request merged**
